foreman/requirements.txt

```diff
@@ -9,15 +9,10 @@ billiard==3.5.0.2         # via celery
 boto==2.46.1
 celery==4.0.2
 django==1.11.1
-geoparse==0.1.10
 kombu==4.0.2              # via celery
-numpy==1.12.1             # via geoparse, pandas
-pandas==0.19.2            # via geoparse
 psycopg2==2.7.1
-python-dateutil==2.6.0    # via pandas
-pytz==2017.2              # via celery, django, pandas
```

Comment:
Nice! I'm glad you commented which dependencies were needed through other dependencies so that they could all be cleaned out in a situation like this one!

Reply:
Well actually, I just reran `pip-compile` on the `requirements.in` file and it automagically regenerated this `requirements.txt` file.

Reply:
Nice :+1: